go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/giantswarm/kubectl-gs
 go 1.16
 
 require (
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
This is required to interface with `github.com/rhysd/go-github-selfupdate`. It was a transitive dependency before, and was linked anyway, so no big difference.